sequencing-server/test/seqjson-to-edsl.spec.ts

```diff
@@ -54,8 +55,49 @@ describe('getEdslForSeqJson', () => {
   });`,
     );
   });
+    */
+
+  it('should throw an error if the user uploads an invalid seqjson', async () => {
+    try {
+      await graphqlClient.request<{
```

Comment:
[Try expect().reject](https://stackoverflow.com/questions/47144187/can-you-write-async-tests-that-expect-tothrow)